model/project_aliases.go

```diff
@@ -83,6 +83,47 @@ func FindAliasesForProjectFromDb(projectID string) ([]ProjectAlias, error) {
 	return out, nil
 }
 
+// MergeAliasesWithProjectConfig returns a merged list of project aliases that includes the merged result of aliases defined
+// on the project ref and aliases defined in the project YAML.  Aliases defined on the project ref will take precedence over the
+// project YAML in the case that both are defined.
+func MergeAliasesWithProjectConfig(projectID string, dbAliases []ProjectAlias) ([]ProjectAlias, error) {
+	dbAliasMap := aliasesToMap(dbAliases)
+	projectConfig, err := FindProjectConfigForProjectOrVersion(projectID, "")
+	if err != nil {
+		return nil, errors.Wrap(err, "error finding project config")
```

Comment:
I think given the changes Kim is going to make to error conventions, errors should now omit the word error, i.e. "finding project config" is sufficient.